### ISSUE TITLE
trouble installing with cython 0.26 (latest)

### ISSUE THREAD

**Issue by neok-m4700** [scikit-image/scikit-image#2729]
I have an issue trying to install version 0.13.0 with latest cython 0.26

```bash
$ python setup.py install --single-version-externally-managed --record record.txt
```
results in:

```python
Error compiling Cython file:
------------------------------------------------------------
...
"""Export fast union find in Cython"""
cimport numpy as cnp

DTYPE = cnp.intp
          ^
------------------------------------------------------------

skimage/measure/_ccomp.pxd:4:11: cimported module has no attribute 'intp'
Compiling <SRC_DIR>/skimage/measure/_ccomp.pyx because it changed.
[1/1] Cythonizing <SRC_DIR>/skimage/measure/_ccomp.pyx
```

numpy 1.13.0
python 3.6.2

**Comment by stefanv** [comment]
I cannot reproduce with the current Git source, Cython 0.26, np 1.13.1, Python 3.6

**Comment by stefanv** [comment]
Also, that line is no longer in the scikit-image source.

**Comment by neok-m4700** [comment]
Allright the issue was in measure/_ccomp.pxd

Fix for v0.13.0 is:
$ sed -i '/DTYPE =/d' skimage/measure/_ccomp.pxd

which has already been fixed in master.

Thanks
